components/item/AwesomeSwiper.vue

```diff
@@ -15,14 +15,16 @@
         class="container h-100"
       >
         <MediaCard
-          :europeana-identifier="item.europeanaIdentifier"
+          :europeana-identifier="europeanaIdentifier"
           :media="item"
+          :is-single-playable-media="isSinglePlayableMedia(item)"
         />
       </div>
       <MediaCard
         v-else
-        :europeana-identifier="item.europeanaIdentifier"
+        :europeana-identifier="europeanaIdentifier"
         :media="item"
+        :is-single-playable-media="isSinglePlayableMedia(item)"
```

Comment:
`isSinglePlayableMedia` no longer accepts an argument. Remove it from this call.